### PR TITLE
new workshop

### PR DESCRIPTION
**Pull request by SallyMcGrath**
New workshop, a very light wrapper around an existing workshop provided by Google. Technical writing 1, all details in readme.

### CODE REVIEW

File: technical-writing-one/readme.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Technical Writing One
+
+Participants: [Pre-class prep, 1 hour](https://developers.google.com/tech-writing/one)
+
+If you haven't done the prep, you can't do this workshop, so go and find something else to get on with.
+
+Facilitators: [Pre-class prep, 1 hour](https://developers.google.com/tech-writing/for-instructors/one/instructors-guide)
```
Comment (wai-t) [inline technical-writing-one/readme.md:7]: Maybe put the Participant and Facilitators instructions into distinct boxes. I find it a little distracting that, at first glance, the two hyperlinks look like they go to the same place labeled "Pre-class prep, 1 hour". I didn't find a pre-existing sibling workshop with a style to match against, so you can use your own judgement.

File: technical-writing-one/readme.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+> Every engineer is also a writer
+
+In this workshop you will practice writing and reviewing technical documentation. This is a critical skill for any engineer. We will use the excellent [Google Technical Writing One](https://developers.google.com/tech-writing/one) workshop materials.
+
+**This workshop will take at least two and a half hours.** Plan to spend a whole morning or whole afternoon on this workshop.
+
+#### Note on AI
+
+Throughout the course, we use Generative AI to help us write and review technical writing. This can be a practical way to get started, but it's important to understand the basic principles and develop your own judgment and clarity. AI frequently generates the _appearance_ of meaning without the substance. This can trick you into thinking your documentation "looks" good when it doesn't actually convey meaning or help anyone understand anything. Don't be Joey:
```
Comment (wai-t) [inline technical-writing-one/readme.md:29]: quotes around "looks" might not be necessary

File: technical-writing-one/readme.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+![Joey improves his writing](./aortic.webp)
+
+## Acceptance Criteria
+
+- We have identified how to address a specific audience
+- We have revised and edited a technical document using the key principles of technical writing
+- We have successfully explained toothbrushes
```
Comment (wai-t) [inline technical-writing-one/readme.md:37]: "toothbrushes"? Buzzword or example from the course materials? I haven't come across it.

**Review by wai-t — APPROVED** [review]
It looks fine as is, but just a couple of suggestions.

File: technical-writing-one/readme.md (hunk 3 above)
Comment (SallyMcGrath) [reply technical-writing-one/readme.md:37]: Oh yeah it's the exercise they complete in the workshop.

**Pull request merged**